tests/baselines/reference/constEnumToStringWithComments.js

```diff
@@ -23,15 +23,15 @@ let c1 = Foo["C"].toString();
 
 
 //// [constEnumToStringWithComments.js]
-var x0 = 100 /* X */..toString();
-var x1 = 100 /* "X" */..toString();
+var x0 = 100 /* X */.toString();

```

Comment:
q: why one of the "." is gone? Also realize that space after number is important 😅